kie-maven-plugin/src/test/java/org/drools/compiler/xpath/tobeinstrumented/InjectReactiveMojoConfigTest.java

```diff
@@ -8,16 +8,20 @@
 
 import org.drools.core.phreak.ReactiveObject;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class InjectReactiveMojoConfigTest {
 
+    private static Logger logger = LoggerFactory.getLogger(InjectReactiveMojoConfigTest.class);
+
     @Test
     public void testRegexpForPackagesDefault() {
         String[] inputConfig = new String[]{"*"};
         
         List<String> config = convertAllToPkgRegExps(inputConfig);
-        
-        System.out.println(Arrays.asList(config));
+
+        logger.info(Arrays.asList(config).toString());
```

Comment:
Just minor thing, but `config` is already list, so no point to convert it to list. `logger.info(config.toString());` should be enough ;) The same applies to the whole file